doc/src/tutorial/index.md

```diff
@@ -25,7 +25,6 @@ Here are some topics that I aim to cover when I get time to write about them:
 - Passing state and type/lifetime parameters to your action code (see e.g.
   [this test](https://github.com/lalrpop/lalrpop/blob/master/lalrpop-test/src/expr_arena.lalrpop)
   invoked [from here]).
-- Location tracking with `@L` and `@R` (see e.g. [this test](https://github.com/lalrpop/lalrpop/blob/master/lalrpop-test/src/intern_tok.lalrpop)).
 - Integrating with external tokenizers (see e.g. [this test](https://github.com/lalrpop/lalrpop/blob/master/lalrpop-test/src/expr.lalrpop)
```

Comment:
Thanks for highlighting that.  I pushed another commit dropping both it and the fallible action line, since that was documented long ago.  Doesn't seem substantial enough to warrant a separate PR.  I think everything else on the list is still a TODO.

Reply:
Thanks, sounds good!